### PR TITLE
Issue 1047 encore docs

### PR DESCRIPTION
**Pull request by wouterboomsma**
Changes made in this Pull Request:
Changes to the documentation of Encore, both factual corrections and edits to improve sphinx rendering.

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

File: package/MDAnalysis/analysis/encore/clustering/affinityprop.pyx [hunk 1]
```diff
@@ -38,23 +38,23 @@ cdef class AffinityPropagation(object):
...
     """
 
     def run(self, s, preference, float lam, int max_iterations, int convergence, int noise=1):
         """
         Run the clustering algorithm.
 
-	    Parameters:
-        ---------
+	Parameters
+        ----------
```
Comment (kain88-de) [inline package/MDAnalysis/analysis/encore/clustering/affinityprop.pyx]: This line doesn't line up with the previous.

File: package/MDAnalysis/analysis/encore/clustering/affinityprop.pyx [hunk 2]
```diff
@@ -70,11 +70,12 @@ cdef class AffinityPropagation(object):
 		    Whether to apply noise to the input s matrix, such there are no equal values. 1 is for yes, 0 is for no.
 		
 
-	    Returns:
-        ---------
+	Returns
+        -------
```
Comment (kain88-de) [inline package/MDAnalysis/analysis/encore/clustering/affinityprop.pyx]: Doesn't line up with the previous line.

File: package/MDAnalysis/analysis/encore/similarity.py [hunk 3]
```diff
@@ -1283,7 +1273,7 @@ def dres(ensembles,
         classes from the dimensionality_reduction module. Different parameters
         for the same method can be explored by adding different instances of
         the same dimensionality reduction class. Provided methods are the
-        Stochastic Proximity Embedding (default) and the Principel Component
+        Stochastic Proximity Embedding (default) and the Principal Component
```
Comment (kain88-de) [inline package/MDAnalysis/analysis/encore/similarity.py]: A side question how does your PCA compare with the new PCA analysis?

File: package/MDAnalysis/analysis/encore/clustering/affinityprop.pyx (hunk 1 above)
Comment (wouterboomsma) [inline package/MDAnalysis/analysis/encore/clustering/affinityprop.pyx]: Tab-vs-space issue. Fixed in e7add80.

File: package/MDAnalysis/analysis/encore/clustering/affinityprop.pyx (hunk 2 above)
Comment (wouterboomsma) [inline package/MDAnalysis/analysis/encore/clustering/affinityprop.pyx]: Tab-vs-space issue. Fixed in e7add80.

File: package/MDAnalysis/analysis/encore/similarity.py (hunk 3 above)
Comment (wouterboomsma) [inline package/MDAnalysis/analysis/encore/similarity.py]: This is a good question. The fact that there are now two versions of PCA in MDAnalysis is a bit of an accident. After early discussions on how encore could best be integrated in MDAnalysis, there was some consensus on making the dimensionality reduction and clustering code more generally available within MDAnalysis. I therefore rewrote the code so that both clustering and dimensionality got their own front-end functions, and that particular clustering and dimensionality reduction algorithms could just be plugged in - for instance those from scikit-learn. PCA was just a natural example to include. Only after doing this work did I realized that someone had coded a PCA in the mean time.

They should probably be merged at some point, but @orbeckst suggested that we stick with the two versions for this release, and then take a step back and think about some refactoring.

So, long story short. I haven't compared the two implementations. But the one in Encore is just a direct interface to the one in scikit-learn.

Comment (kain88-de) [inline package/MDAnalysis/analysis/encore/similarity.py]: So then you need to load everything into memory while the other implementation only needs to hold the current frame and can therefore estimate a PCA for even a TB trajectory.

Comment (wouterboomsma) [inline package/MDAnalysis/analysis/encore/similarity.py]: Right. That is at least one obvious difference.
